.github/workflows/build.yaml

```diff
@@ -254,27 +273,35 @@ jobs:
           cd third_party\OpenDDS
           configure
 
-      - name: Download OpenDDS Binaries
-        shell: cmd
-        run: |
-          python %GITHUB_WORKSPACE%\third_party\CommonTools\scripts\utilsftp.py -c download -s ${{ secrets.FTP_STORE_SITE }} -u ${{ secrets.FTP_STORE_USER }} -p ${{ secrets.FTP_STORE_PASSWORD }} -r OpenDDS_Build_3.25.zip -l %GITHUB_WORKSPACE%\OpenDDS_Build_3.25.zip
-        env:
-          GITHUB_WORKSPACE: $GITHUB_WORKSPACE
-          
-      - name: Download OpenDDS Headers
-        shell: cmd
-        run: |
-          python %GITHUB_WORKSPACE%\third_party\CommonTools\scripts\utilsftp.py -c download -s ${{ secrets.FTP_STORE_SITE }} -u ${{ secrets.FTP_STORE_USER }} -p ${{ secrets.FTP_STORE_PASSWORD }} -r OpenDDS_Headers_3.25.zip -l %GITHUB_WORKSPACE%\OpenDDS_Headers_3.25.zip
-        env:
-          GITHUB_WORKSPACE: $GITHUB_WORKSPACE
+      - name: Restore OpenDDS from cache
+        uses: actions/cache/restore@v3
+        with:
+          fail-on-cache-miss: true
+          enableCrossOsArchive: true
+          path: ./OutputOpenDDS/
+          key: OpenDDS_3.25
+
+    #  - name: Download OpenDDS Binaries
+    #    shell: cmd
+    #    run: |
+    #      python %GITHUB_WORKSPACE%\third_party\CommonTools\scripts\utilsftp.py -c download -s ${{ secrets.FTP_STORE_SITE }} -u ${{ secrets.FTP_STORE_USER }} -p ${{ secrets.FTP_STORE_PASSWORD }} -r OpenDDS_Build_3.25.zip -l %GITHUB_WORKSPACE%\OpenDDS_Build_3.25.zip
+    #    env:
+    #      GITHUB_WORKSPACE: $GITHUB_WORKSPACE
+    #      
+    #  - name: Download OpenDDS Headers
+    #    shell: cmd
+    #    run: |
+    #      python %GITHUB_WORKSPACE%\third_party\CommonTools\scripts\utilsftp.py -c download -s ${{ secrets.FTP_STORE_SITE }} -u ${{ secrets.FTP_STORE_USER }} -p ${{ secrets.FTP_STORE_PASSWORD }} -r OpenDDS_Headers_3.25.zip -l %GITHUB_WORKSPACE%\OpenDDS_Headers_3.25.zip
+    #    env:
+    #      GITHUB_WORKSPACE: $GITHUB_WORKSPACE
```

Comment:
Clean code

Reply:
Leaved in workflow till completed at least a first release cycle